src/Form/ValidateEmailForm.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\dpc_user_management\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+class ValidateEmailForm extends FormBase{
+    /**
+     * Returns a unique string identifying the form.
+     *
+     * The returned ID should be a unique string that can be a valid PHP function
+     * name, since it's used in hook implementation names such as
+     * hook_form_FORM_ID_alter().
+     *
+     * @return string
+     *   The unique string identifying the form.
+     */
+    public function getFormId() {
+        return 'validate_email_form';
+    }
+
+
+    /**
+     * @param array $form
+     * @param FormStateInterface $form_state
+     * @return array
```

Comment:
```suggestion
     * @param FormStateInterface $form_state
     * @param $parameter (optional)
     * @return array
```

🐘

Reply:
Interface actually has only 2 parameters and `$parameter` is not used in the form either.